modules/tensor_mechanics/test/tests/tensile/tests

```diff
@@ -12,7 +12,6 @@
     csvdiff = 'small_deform1.csv'
     rel_err = 1.0E-5
     abs_zero = 1.0E-5
-    cli_args = 'GlobalParams/volumetric_locking_correction=true'
```

Comment:
Why are you removing all those? This does not look like parser related, but an actual physics change.

Reply:
Unused parameters.

Reply:
volumetric locking correction is false by default in tensor mechanics